internal/pkg/vm/actor/builtin/miner/miner.go

```diff
@@ -1085,65 +1034,67 @@ func (a *Impl) SubmitPoSt(ctx invocationContext, poStProof types.PoStProof, faul
 		//
 		// This switching will be removed when issue #2270 is completed.
 		if !a.Bootstrap {
-			// calculate challenge seed now. If PoSt is too early, this will fail.
-			poStChallengeTime := types.NewBlockHeight(PoStChallengeWindowBlocks)
-
-			var seed types.PoStChallengeSeed
-			var provingWindowStart, provingWindowEnd *types.BlockHeight
-			if chainHeight.LessThan(state.ProvingPeriodEnd) {
-				// proof is in time, sample at start of post challenge time
-				provingWindowStart = state.ProvingPeriodEnd.Sub(poStChallengeTime)
-				provingWindowEnd = state.ProvingPeriodEnd
-			} else {
-				// proof is late, sample at start of next post challenge time
-				provingWindowStart = nextProvingPeriodEnd.Sub(poStChallengeTime)
-				provingWindowEnd = nextProvingPeriodEnd
-			}
-
-			if chainHeight.LessThan(provingWindowStart) {
-				return nil, errors.NewCodedRevertErrorf(ErrInvalidPoSt, "PoSt arrived at %s, which is before proving window (%s-%s)",
-					chainHeight.String(), provingWindowStart.String(), provingWindowEnd.String())
-			}
-
-			seed, err = getPoStChallengeSeed(ctx, state, provingWindowStart)
-			if err != nil {
-				return nil, errors.RevertErrorWrap(err, "failed to sample chain for challenge seed")
-			}
-
-			var sectorInfos []go_sectorbuilder.SectorInfo
-			for _, id := range state.ProvingSet.Values() {
-				commitment, found := state.SectorCommitments.Get(id)
-				if !found {
-					return nil, errors.NewFaultErrorf("miner ProvingSet sector id %d missing in SectorCommitments", id)
-				}
-				sectorInfo := go_sectorbuilder.SectorInfo{
-					SectorID: id,
-					CommR:    commitment.CommR,
-				}
-				sectorInfos = append(sectorInfos, sectorInfo)
-			}
-
-			sortedSectorInfo := go_sectorbuilder.NewSortedSectorInfo(sectorInfos...)
-			log.Infof("Verifying post for addr %s -- end: %s -- seed: %x", ctx.LegacyMessage().To, state.ProvingPeriodEnd, seed)
-			for i, ssi := range sortedSectorInfo.Values() {
-				log.Infof("ssi %d: sector id %d -- commR %x", i, ssi.SectorID, ssi.CommR)
-			}
+			panic("this needs to use new sectorbuilder")
 
-			req := verification.VerifyPoStRequest{
-				ChallengeSeed:    seed,
-				SortedSectorInfo: sortedSectorInfo,
-				Faults:           faults.SectorIds.Values(),
-				Proof:            poStProof,
-				SectorSize:       state.SectorSize,
-			}
-
-			res, err := ctx.LegacyVerifier().VerifyPoSt(req)
-			if err != nil {
-				return nil, errors.RevertErrorWrap(err, "failed to verify PoSt")
-			}
-			if !res.IsValid {
-				return nil, Errors[ErrInvalidPoSt]
-			}
+			// calculate challenge seed now. If PoSt is too early, this will fail.
```

Comment:
Just delete this.  If actors people ever want to look at how we used to do post in gfc they can go spelunking through history.

Reply:
@acruikshank to handle this